### DISCUSSION TITLE
Buildable library dependency

### DISCUSSION THREAD

**Discussion by GeorgianStan** [nrwl/nx#19495] [General]
Hi!
I have an Angular library that I want to convert to a buildable library. For that, I've followed the official [guide](https://nx.dev/recipes/angular/setup-incremental-builds-angular).

First, I generated a new buildable library and then copied and pasted the source code from the original library into the new library.
The `package.json` file of the new buildable library that was generated by NX looks like this:
```json
{
  "name": "@my-org/my-angular-buildable-library",
  "version": "0.0.1",
  "peerDependencies": {
    "@angular/common": "^16.0.0",
    "@angular/core": "^16.0.0"
  },
  "dependencies": {
    "tslib": "^2.3.0"
  },
  "sideEffects": false
}
```
This library, `my-angular-buildable-library`, uses some npm packages as dependencies. Do these dependencies need to be manually added to the generated `package.json` file under the `dependencies` property?

**Comment by sudo-vaibhav** [comment]
any answer to this?
